### PR TITLE
Change MpTelemetry-2.0 tests run in LITE mode

### PR DESCRIPTION
**Pull request by Yas-Franco**
For issue #28766

### CODE REVIEW

File: dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/internal/suite/FATSuite.java [hunk 1]
```diff
@@ -80,7 +80,7 @@ public static RepeatTests telemetry10and11Repeats(String serverName) {
     }
 
     public static RepeatTests telemetry20Repeats(String serverName) {
-        return TelemetryActions.repeat(serverName, TelemetryActions.MP14_MPTEL20, TelemetryActions.MP41_MPTEL20, TelemetryActions.MP50_MPTEL20, MicroProfileActions.MP70_EE10, MicroProfileActions.MP70_EE11);
+        return TelemetryActions.repeat(serverName, MicroProfileActions.MP70_EE11, TelemetryActions.MP14_MPTEL20, TelemetryActions.MP41_MPTEL20, TelemetryActions.MP50_MPTEL20, MicroProfileActions.MP70_EE10, MicroProfileActions.MP70_EE11);
```
Comment (tevans78) [inline dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/internal/suite/FATSuite.java]: You're running MP70_EE11 twice, both LITE and FULL

**Comment by Yas-Franco** [comment]
#libby
#build

**Comment by Yas-Franco** [comment]
The defects in the build are from an issue that should be fixed now. Running another build to be sure

#build

**Comment by Yas-Franco** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck

**Comment by Yas-Franco** [comment]
#libby

**Review by tevans78 — APPROVED** [review]

**Pull request merged**